gapic/schema/wrappers.py

```diff
@@ -383,14 +395,59 @@ def oneof_fields(self, include_optional=False):
 
         return oneof_fields
 
+    @utils.cached_property
+    def extended_operation_request_fields(self) -> Sequence[Field]:
```

Comment:
How is this different than `operation_request_field`? My question is mostly born from not understanding LRO and extended LRO. Like, what is the relationship between these two methods here.

Reply:
This is a method on `Message`; `operation_request_field` is a method on `Field`. They are complementary: for a message, we want to know all the fields that are `operation_request_fields`, and for a field, we want to know which field in the operation request message it maps to.